src/Integration.Vsix/Settings/GeneralOptionsDialogControl.xaml.cs

```diff
@@ -32,96 +29,16 @@ namespace SonarLint.VisualStudio.Integration.Vsix
     /// </summary>
     public partial class GeneralOptionsDialogControl : UserControl
     {
-        private readonly ISonarLintSettings settings;
```

Comment:
Removed the code that showed/hid the controls depending on the state of the text box. That meant the control no longer needs the settings instance or the logger.